jans-cedarling/cedarling/src/authz/entities/test_create.rs

```diff
@@ -263,16 +264,16 @@ fn get_token_claim_type_boolean_error() {
     if let CedarPolicyCreateTypeError::GetTokenClaimValue(GetTokenClaimValue::KeyNotCorrectType {
         key,
         got_type,
-        ..
+        expected_type,
     }) = entity_creation_error
     {
         let json_attr_value = json.as_object().unwrap().get(test_key).unwrap();
         let origin_type = GetTokenClaimValue::json_value_type_name(json_attr_value);
 
-        assert!(key == test_key, "expected key: {test_key}, but got: {key}");
+        assert!(key == test_key, "expected key: {test_key}, but got: {key} with schema expected_type {expected_type}");
```

Comment:
Just a minor change to keep the consistency. 
Add ":" after expected_type
`"expected` key: {test_key}, but got: {key} with schema expected_type: {expected_type}"`

Reply:
I would agree with your point on consistency if this was a log string, or a display string, or something that was output in the normal course of running the code.

But it's just an error string in case this test fails sometime in the future. Not sure it's worthwhile making an issue of something like that.

But I'll change it.